go/tools/gazelle/gazelle/main.go

```diff
@@ -69,11 +77,26 @@ func run(dirs []string, emit func(*bzl.File) error) error {
 		}
 		for _, f := range files {
 			f.Path = filepath.Join(*repoRoot, f.Path)
-			if f, err = merger.MergeWithExisting(f); err != nil {
-				return err
-			}
-			bzl.Rewrite(f, nil) // have buildifier 'format' our rules.
-			if err := emit(f); err != nil {
+			existingFilePath, err := findBuildFile(path.Dir(f.Path))
+			if os.IsNotExist(err) { // No existing file, so write a new one
+				bzl.Rewrite(f, nil) // have buildifier 'format' our rules.
+				if err := emit(f); err != nil {
+					return err
+				}
+			} else if err == nil { // Existing file, so merge and maybe remove the old one
```

Comment:
Use continue and guard-clauses, so add a continue at the end of the above if loop, 
  continue
}
if err != nil {
  return err
}
this code w/ no indent.